code/modules/mob/living/silicon/say.dm

```diff
@@ -26,7 +26,7 @@
 				var/mob/living/silicon/ai/ai = src
 				following = ai.eyeobj
 			var/link = FOLLOW_LINK(M, following)
-			to_chat(M, "[link] [rendered]")
+			to_chat(M, "<span class='binarysay'>[link] [rendered]</span>")

```

Comment:
Won't this colour the blue (F)?

Reply:
Yeah but it would look out of place anyways with the black and green.